### PR TITLE
Store vic machine server logs in log disk

### LINKED ISSUE #944: store vic-machine-server logs on the appliance
**User Statement:**

As a user I want to manage vic-machine-server logs.

**Details:**
Depends on https://github.com/vmware/vic/issues/6571

**Acceptance Criteria:**
- [ ] Update service to mount volume and put logs in the right place
- [ ] Able to configure log rotation or set a max log size

### PR DESCRIPTION
**Pull request by morris-jason**
Fixes #944 by:
 - adding a logrotate config to the appliance
 - adding a volume mount to the vic-machine-server service

### CODE REVIEW

File: installer/build/scripts/systemd/vic_machine_server/vic-machine-server-logrotate [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/storage/log/vic-machine-server/vic-machine-server.log {
+    daily
+    size 1G
+    rotate 10
+    compress
+    delaycompress
+    missingok
+    notifempty
+    create 644 root root
```
Comment (zjs) [inline installer/build/scripts/systemd/vic_machine_server/vic-machine-server-logrotate]: Is `root` the correct owner? (Should UID 10000 be used?)

Comment (andrewtchin) [reply installer/build/scripts/systemd/vic_machine_server/vic-machine-server-logrotate]: i havent updated the UID for vic-machine-server to 10000 yet, i was waiting until the feature branch is merged

File: installer/build/scripts/systemd/vic_machine_server/vic-machine-server-logrotate [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/storage/log/vic-machine-server/vic-machine-server.log {
+    daily
+    size 1G
+    rotate 10
+    compress
+    delaycompress
+    missingok
+    notifempty
```
[the remark this answers is not in the document]
Comment (morris-jason) [reply installer/build/scripts/systemd/vic_machine_server/vic-machine-server-logrotate]: My idea was the opposite of yours - I don't want to see dates rotated log files, implying we had logs for those days, that are empty.
Let me know if you'd like to switch it.

[the remark this answers is not in the document]
Comment (morris-jason) [reply installer/build/scripts/systemd/vic_machine_server/vic-machine-server-logrotate]: Decided to go with Zach's approach. fixed.

File: installer/build/scripts/systemd/vic_machine_server/vic-machine-server-logrotate (hunk 1 above)
Comment (morris-jason) [reply installer/build/scripts/systemd/vic_machine_server/vic-machine-server-logrotate]: fixed.

**Pull request merged**